### PR TITLE
Add tests and samples to anomaly detector

### PR DESCRIPTION
**Pull request by luxiong-ms**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by pshao25** [comment]
What is the change for sdk/anomalydetector/Azure.AI.AnomalyDetector/src/Generated/Models/TimeSeriesPoint.Serialization.cs?

Please add the changelog.

**Comment by luxiong-ms** [comment]
> What is the change for sdk/anomalydetector/Azure.AI.AnomalyDetector/src/Generated/Models/TimeSeriesPoint.Serialization.cs?
> 
> Please add the changelog.

There is no change in sdk/anomalydetector/Azure.AI.AnomalyDetector/src/Generated/Models/TimeSeriesPoint.Serialization.cs. 
changelog is added.

**Review by pshao25 — APPROVED** [review]

File: sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample3_DetectChangePoint.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+using Azure.AI.AnomalyDetector.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.AnomalyDetector.Tests.Samples
+{
+    public partial class AnomalyDetectorSamples : SamplesBase<AnomalyDetectorTestEnvironment>
+    {
+        [Test]
+        public void DetectChangePoint()
```
Comment (ArcturusZhang) [inline sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample3_DetectChangePoint.cs:20]: Should we mark the sample with attribute `[Ignore]` to skip them? Or the samples here are actually runnable?

Comment (luxiong-ms) [reply sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample3_DetectChangePoint.cs:20]: Sample tests in this folder are runnable.

**Pull request merged**